firmware/config/config_helper.h

```diff
@@ -1,6 +1,5 @@
-#if !__has_include("config.h")
-    #error "You need to copy firmware/config/config.h.template to firmware/config/config.h and modify it to match your configuration"
-#else
+// Include config.h if it exists
+#ifdef USE_CONFIG_H
```

Comment:
```suggestion
#if __has_include("config.h")
```
This should do it automatically?

No need to specify the `USE_CONFIG_H` somewhere.